src/storage/lock_manager/deadlock.rs

```diff
@@ -160,6 +167,26 @@ impl DetectTable {
     {
         self.wait_for_map.retain(|ts, _| !is_expired(*ts));
     }
+
+    /// The threshold of detect table size to trigger `active_expire`.
+    const ACTIVE_EXPIRE_THRESHOLD: usize = 100000;
+    /// The interval between `active_expire`.
+    const ACTIVE_EXPIRE_INTERVAL: Duration = Duration::from_secs(60);
```

Comment:
May this be too frequent? I think it's ok to set it much longer.

Reply:
An hour or a day?

Reply:
I changed it to an hour.